### PR TITLE
feat: `ParaToSystem`

### PR DESCRIPTION
**Pull request by TarikGul**
TODO:

- [x] Support for `GeneralKey` assets for erc20 tokens.
    - [X] Add GeneralKey MultiLocation generation
    - [x] Ensure sorting assets is correct for GeneralKeyInterior type.
- [X] Error validation layer 
    - [x] Create an issue for a part 2 where we expand on erc20 token addresses to be added to the registry.
- [x] Unit Tests for `ParaToSystem`
- [x] Integration tests for `ParaToSystem`
- [x] Docs

This might also require updating the registry to include assetId's that point to their respective tokens across chains.

closes: https://github.com/paritytech/asset-transfer-api/issues/192

### CODE REVIEW

**Comment by TarikGul** [comment]
https://github.com/paritytech/asset-transfer-api-registry/pull/7 is required for setting up a MultiAssets for `ParaToSystem`

File: src/integrationTests/AssetsTransferApi.spec.ts [hunk 1]
```diff
@@ -808,6 +814,141 @@ describe('AssetTransferApi Integration Tests', () => {
...
 			});
 		});
+		describe('ParaToSystem', () => {
+			const baseParachainCreateTx = async <T extends Format>(
+				format: T,
+				isLimited: boolean,
+				xcmVersion: number
+			): Promise<TxResult<T>> => {
+				return await moonbeamAssetsApi.createTransferTransaction(
+					'1000', // `0` indicating the dest chain is a relay chain.
```
[the remark this answers is not in the document]
Comment (TarikGul) [reply src/integrationTests/AssetsTransferApi.spec.ts:824]: Ahh yes, thanks!

**Review by marshacb — APPROVED** [review]
looks great to me! just a minor nit

File: src/AssetsTransferApi.spec.ts [hunk 2]
```diff
@@ -22,6 +23,11 @@ const systemAssetsApi = new AssetsTransferApi(
 	2
 );
 const relayAssetsApi = new AssetsTransferApi(adjustedMockRelayApi, 'kusama', 2);
+const moonbeamAssetsApi = new AssetsTransferApi(
```
Comment (marshacb) [inline src/AssetsTransferApi.spec.ts:26]: we intentionally name this moonbeam rather than moonriver?

Comment (TarikGul) [reply src/AssetsTransferApi.spec.ts]: Nice I switched it all to moonriver. Nice catch

Comment (TarikGul) [reply src/AssetsTransferApi.spec.ts]: fixed

**Pull request merged**